### DISCUSSION TITLE
Typo in spherical harmonics formula

### DISCUSSION THREAD

**Discussion by wumo** [google/filament#5844] [Ideas]
[The formula for SH](https://google.github.io/filament/Filament.html#annex/sphericalharmonics/basisfunctions)

$$
\begin{align*} y^{m < 0}_l &= \sqrt{2} K^m_l sin(m \phi) P^{\|m\|}_l(cos \theta) \end{align*}
$$

should be

$$
\begin{align*} y^{m < 0}_l &= \sqrt{2} K^m_l sin(\|m\| \phi) P^{\|m\|}_l(cos \theta) \end{align*}
$$

**Comment by romainguy** [comment]
@pixelflinger

**Comment by pixelflinger** [comment]
Yes, thank you for catching this typo.
